CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+##2015-08-25 - Supported Release 1.7.1
+###Summary
+
+This is a bugfix release to deprecate the port parameter, using port can potentially create a security issue. Use dport and/or sport instead.
```

Comment:
Please explain/reference an explanation of the security issue, so that the reader can understand the problem and evaluate counter-measures.